tutorials/understanding-group/index.md

```diff
@@ -178,7 +194,7 @@ This last command outputs `1` too. This shows you that the group and its `id` ca
 $ simd query group group-members $GROUP_ID
 ```
 
-Nice! Your group has `best football association` as metadata (which you can recall with the `group-info` command), Alice as group admin, and Alice and Bob as group members.
+Nice! Your group has `QmbZAqeGApZTfFkVEL3SL1NG5no37B6b6bbkw5g7z5Pzga` as metadata (which you can recall with the `group-info` command), Alice as group admin, and Alice and Bob as group members.
```

Comment:
```suggestion
Nice! Your group has `QmbZAqeGApZTfFkVEL3SL1NG5no37B6b6bbkw5g7z5Pzga` as metadata (which you can recall with the `group-info` command), Alice as group admin, and Alice and Bob as group members.

<HighlightBox type="note">

The members of your group understand that this hash is for IPFS. Now, if you want to make it intelligible to a wider audience, you could replace that with:

* `ipfs://QmbZAqeGApZTfFkVEL3SL1NG5no37B6b6bbkw5g7z5Pzga`, which, although more understandable, is not a standard but ought to be.
* Or `https://ipfs.io/ipfs/QmbZAqeGApZTfFkVEL3SL1NG5no37B6b6bbkw5g7z5Pzga` which is a valid URL.

</HighlightBox>
```

Reply:
I can add, but not sure about that. It depends on the support of the clients. @hxrts it is not clear from the metadata spec what the recommendation is. Would you know?

Reply:
ipfs:// is a pretty common convention and I think it make sense to recommend use

Reply:
Alright, added!